test/badge/minting.ts

```diff
@@ -104,20 +107,41 @@ describe("Badge - Minting", function () {
         }));
       });
 
-      it("should revert with an invalid signature", async () => {
+      it("should revert with an invalid signer", async () => {
         await expect(
           badgeMint
             .connect(admin)
             .mintToEOA(ethers.constants.AddressZero, tokenId, mintAmount, v, r, s, tokenURI, data),
         ).to.be.revertedWith("ERC1238: Approval verification failed");
       });
 
+      // TO-DO properly test ecrecover to address(0x0)?
+      // it("should revert with an invalid signature", async () => {
+      //   await expect(
+      //     badgeMint
+      //       .connect(admin)
+      //       .mintToEOA(ethers.constants.AddressZero, tokenId, mintAmount, v, r, s, tokenURI, data),
+      //   ).to.be.revertedWith("ERC1238: invalid signature");
+      // });
+
```

Comment:
@ra-phael I'm not entirely sure how to make a test that causes the `ecrecover` to return the `0` address. Any ideas?